shared/team-building/index.tsx

```diff
@@ -495,12 +499,22 @@ class TeamBuilding extends React.PureComponent<Props, {}> {
             onRedoRecs={props.fetchUserRecs}
           />
         )}
-        {this._searchInput()}
-        {this._listBody()}
-        {props.waitingForCreate && (
-          <Kb.Box2 direction="vertical" style={styles.waiting} alignItems="center">
-            <Kb.ProgressIndicator type="Small" white={true} style={styles.waitingProgress} />
-          </Kb.Box2>
+        {props.selectedService === 'phone' ? (
```

Comment:
If you wanna help make a future conflict a little less painful, I made this logic into a switch statement starting at 454. My code:
```
    let content
    switch (props.selectedService) {
      case 'email':
        content = <EmailInput />
        break
      // case 'phone':
      //   break
      default:
        content = (
          <>
            {this._searchInput()}
            {this._listBody()}
            {props.waitingForCreate && (
              <Kb.Box2 direction="vertical" style={styles.waiting} alignItems="center">
                <Kb.ProgressIndicator type="Small" white={true} style={styles.waitingProgress} />
              </Kb.Box2>
            )}
          </>
        )
    }
```

Reply:
@nathunsmitty Can you link me to your code where you did this? Without switch expressions I don't see how what you're doing works. Are you abstracting it into a separate method?